lib/metadata/labels.js

```diff
@@ -45,7 +45,10 @@ var LABEL_TO_PATTERNS = {
         "tools/**"
     ],
     "testdriver.js": ["resources/testdriver*"],
-    "testharness.js": ["resources/testharness*", "resources/test/**"]
+    "testharness.js": ["resources/testharness*", "resources/test/**"],
+    "Azure Pipelines": [".azure-pipelines.yml", "tools/ci/azure/**"],
+    "Taskcluster": [".taskcluster.yml"],
+    "Travis CI": [".travis.yml"]
```

Comment:
This line is no longer applicable as of https://github.com/web-platform-tests/wpt/pull/16086

Reply:
I pushed a commit to remove this line.